g2p/tests/test_transducer.py

```diff
@@ -209,6 +210,42 @@ def test_deletion(self):
         self.assertEqual(self.test_deletion_transducer_csv("a").output_string, "")
         self.assertEqual(self.test_deletion_transducer_json("a").output_string, "")
 
+    def test_case_preservation(self):
+        mapping = Mapping(
+            [
+                {"in": "'a", "out": "b"},
+                {"in": "e\u0301", "out": "f"},
+                {"in": "tl", "out": "λ"},
+            ],
+            case_sensitive=False,
+            preserve_case=True,
+            norm_form="NFC",
+            case_equivalencies={"λ": "\u2144"},
+        )
+        transducer = Transducer(mapping)
+        self.assertEqual(transducer("'a").output_string, "b")
+        self.assertEqual(transducer("'A").output_string, "B")
+        self.assertEqual(transducer("E\u0301").output_string, "F")
+        self.assertEqual(transducer("E\u0301").output_string, "F")
+        # Test what happens in Heiltsuk. \u03BB should be capitalized as \u2144
```

Comment:
We can help the reader of the code by inserting the rendered characters
```suggestion
        # Test what happens in Heiltsuk. \u03BB (λ) should be capitalized as \u2144 (⅄)
```